docs/getting_started/Demo.md

```diff
@@ -0,0 +1,347 @@
+
+Create a Github issue and label it appropriately to register a failure, like in the following image.
+
+![github_issue1](../img/github_issue_1.png)
+
+Refresh the Grafana dashboard and you should see the Change Failure Rate go up.
+
+![change_failure_rate_1](../img/change_failure_rate_1.png)
+(did not went up for me)
+
+Now, let's create a non critical bug: a bug that does not indicate a deployment failure in your todolist application.
+
+Create another Github issue and add the bug label, however do **not** add the application label.
+
+![issue_untagged](../img/issue_2_non_deployment.png)
+
+Now, let's resolve issue #1 and see how that impacts our `Failure Rate` and the `Mean Time to Restore`.
+
+![issue_close](../img/github_issue_1_close.png)
+
+Check the output from the failure exporter again, by running
+```
+curl $(oc get route failure-exporter --namespace pelorus -o=template='http://{{.spec.host | printf "%s\n"}}')
+```
+Issue #1 should be found in the output of the curl. Issue #2 will not be registered as a deployment failure because the issue is **not** tagged with `app.kubernetes.io/name=todolist`.
+
+Notice the fields `failure_creation_timestamp` and `failure_resolution_timestamp` in the previous command's output. They indicate the time the issue was created and when it was closed.
+
+### Understand the changes to the Grafana Dashboard
+
+Now, we should also have data in the `Mean Time to Restore` metric in Grafana dashboard, like in the following image.
+
+![mean_time_to_restore](../img/mean_time_to_restore.png)
+
+**Mean Time to Restore** measures how long it takes to restore the service when an incident occurs.
+
+**Change Failure Rate** is a key quality metric that measures what percentage of changes fails in the production environment. It is crucial to have alignment on what constitutes a failure. The recommended definition is a change that either results in degraded service or subsequently requires remediation.
+
+## Partially Automated Demo
+
+The Pelorus and todolist application can be installed automatically. Using the forked copy of [mig-demo-apps](https://github.com/konveyor/mig-demo-apps) referenced as `https://github.com/<your_org>/mig-demo-apps`, execute the following steps:
+
+* [**Optional step**] Setup the [GitHub webhook](#github-webhook).
+
+* Enable Github Issues in the repository's settings and create at least one Github issue with the `bug` and `app.kubernetes.io/name=todolist` issue labels.
+
+* Execute `run-pelorus-e2e-tests`
+```
+cd pelorus
+export KUBECONFIG=$PATH_TO_KUBECONFIG_FILE
+export TOKEN=<github_personal_access_token>
+make dev-env
+source .venv/bin/activate
+scripts/run-pelorus-e2e-tests -o <your_org> -e failure
+```
+(does it need to export kubeconfig and run make dev env and run inside virtual environment?)
+(TOO MUCH NOISE!!!)
```

Comment:
I felt that script outputs too much information, I would try to suppress some logs